tests/integration_tests/test_lightgbm.py

```diff
@@ -6,16 +7,25 @@
 from optuna.testing.integration import DeterministicPruner
 
 
-def test_lightgbm_pruning_callback_call():
-    # type: () -> None
+# If `True`, `lgb.cv(..)` will be used in the test, otherwise `lgb.train(..)` will be used.
+CV_FLAGS = [False, True]
 
-    env = lgb.callback.CallbackEnv(
-        model='test',
-        params={},
-        begin_iteration=0,
-        end_iteration=1,
-        iteration=1,
-        evaluation_result_list=[('validation', 'binary_error', 1., False)])
+
+@pytest.mark.parametrize('cv', CV_FLAGS)
+def test_lightgbm_pruning_callback_call(cv):
+    # type: (bool) -> None
+
+    callback_env = partial(lgb.callback.CallbackEnv,
+                           model='test',
+                           params={},
+                           begin_iteration=0,
+                           end_iteration=1,
+                           iteration=1)
+
+    if cv:
+        env = callback_env(evaluation_result_list=[(('cv_agg', 'binary_error', 1., False, 1.))])
+    elif not cv:
```

Comment:
I think `else` is more concise in this case.
The same is true for https://github.com/pfnet/optuna/pull/260/files#diff-b106f266d7dc791aa59a717e323c11fcR107.